sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -482,6 +482,15 @@ case class JsonTuple(children: Seq[Expression])
 /**
  * Converts an json input string to a [[StructType]] with the specified schema.
  */
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = "_FUNC_(jsonStr, schema[, options]) - Return a `StructType` value with the given `jsonStr` and `schema`.",
```

Comment:
Probably just `struct` instead of `` `StructType` `` (as I found a example in https://github.com/apache/spark/blob/e7f982b20d8a1c0db711e0dcfe26b2f39f98dd64/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala#L340 as a reference).